announcements/1.18.1.md

```diff
@@ -13,7 +13,8 @@ Please see [component versions](https://anbox-cloud.io/docs/component-versions)
 
 ## Bug fixes
 
-< List bugs that were part of the current release. Include JIRA or launchpad IDs and links for the bugs >
+- [AC-1560](https://warthogs.atlassian.net/browse/AC-1560) - `amc start <container_id>` now asks for a confirmation to avoid accidental log deletion for an errored container.
+- [AC-1570](https://warthogs.atlassian.net/browse/AC-1570) - An intuitive error message is shown in cases where `amc application create` fails due to snap access issues.
```

Comment:
It might be worth detailing one or two lines about the bugs. Sorry the template is a little outdated. I will correct it to reflect our current practice. 
Add Jira or LP links for customer reported bugs and for the rest, just describe the fix in 1-2 lines.